doc/reference/native-tokens/02-getting-started.md

```diff
@@ -0,0 +1,582 @@
+
+### Configuring cardano-node to run using cardano-cli
+
+*This document describes how to interact with the node from a bash command line, but it is also possible to download and run the node as a `docker` image, which will involve less system configuration. The docker image is `input-output/cardano-node` - please ensure you have the [latest version](https://github.com/input-output-hk/cardano-node/releases) of the node.*
+
+To configure cardno-node to communicate with the public testnet environment, first build and install the node as described in [https://github.com/input-output-hk/cardano-node](https://github.com/input-output-hk/cardano-node), and then set up its configuration:  
+
+```
+mkdir -p testnet/config
+cd testnet
+# download and save the configuration files to the config subdirectory
+```
+
+You may then use the following command from the `mary-ppe` directory to start the relay node. Note that you **do not need** to configure and run the node as a “stake pool” (block producing node):
+
+```
+export CARDANO_NODE_SOCKET_PATH=socket
+ 
+cardano-node run --topology config/topology.json  
+--database-path db --config config/config.json --port 3001 
+--socket-path "$CARDANO_NODE_SOCKET_PATH"
+```
+
+The latest configuration files can be found [here](https://github.com/input-output-hk/cardano-world/tree/master/docs/environments). We recommend you download all the files that are associated with the `pre-prod` cluster and copy them to your `config` subdirectory.
+
+Once you have started the node, leave it running in its own terminal window (or run it in the background if you prefer).
+
+### Using the Cardano CLI
+
+Once a relay node has been set up, the `cardano-cli` command can be used to interact with the network as usual: 
+
+```
+cd testnet
+cardano-cli ...
+```
+
+### Understanding values
+
+cardano-node version 1.32.1 and later of the cli no longer supports ASCII token names, only hex-encoded format is supported. The result of running  
+
+```
+$ echo -n "assetName" | xxd -ps
+```
+
+Lovelace values can be specified in two ways:
+
+- `${quantity} lovelace` (where quantity is a signed integer)
+- `${quantity}` (where quantity is a signed integer)
+- `${assetName}` (where assetName is hex-encoded 61737365744e616d65)
+
+Values for other assets can be specified as:
+
+- `${quantity} ${policyId}.${assetName}`
+- `${quantity} ${policyId}` 
+
+Where `quantity` is a signed integer and `policyId` is a hex-encoded policy ID [a script hash]), and `assetName` is a hex-encoded assetName.
+
+#### Syntax of multi-asset values
```

Comment:
A lot of this document covers things spoken about in `doc/reference/multi-assets.md`. Please compare both and consolidate the two into a single document.

Reply:
Done